provisioning-workflows/IIB/IIBv9/jcl/AUTHFS.jcl

```diff
@@ -0,0 +1,43 @@
+//S1     EXEC USSBATCH,SU=Y
```

Comment:
USSBATCH is not a standard utility available on z/OS, maybe we should find a more standard approach.

Reply:
I agree. Do we have to wait for Niek to make change or @RonaldKristel will you be able to make this change?

Reply:
USSBATCH is now supplied as seperate artifact in workflow folder.